controllers/etcd_controller.go

```
@@ -406,59 +348,81 @@ func (r *EtcdReconciler) delete(ctx context.Context, etcd *druidv1alpha1.Etcd) (
 
 func (r *EtcdReconciler) reconcilePodDisruptionBudget(ctx context.Context, logger logr.Logger, etcd *druidv1alpha1.Etcd, values map[string]interface{}) error {
 	logger.Info("Reconcile PodDisruptionBudget")
-	pdb := &policyv1beta1.PodDisruptionBudget{}
-	err := r.Get(ctx, types.NamespacedName{Name: etcd.Name, Namespace: etcd.Namespace}, pdb)
 
-	if err == nil {
-		// pdb already exists, claim it
+	// Find out k8s version
+	targetCoreClient, _ := k8s.NewForConfig(r.Config)
```

You could return the error.